app/scripts/modules/kubernetes/src/v2/pipelines/stages/undoRolloutManifest/undoRolloutManifestConfig.html

```diff
@@ -0,0 +1,26 @@
+<div class="form-horizontal">
+  <div class="alert alert-warning">
+    <strong>Warning!</strong> This stage is under active development and is subject to change.
+  </div>
+  <h4>Manifest</h4>
+  <div class="horizontal-rule"></div>
+  <kubernetes-manifest-selector selector="ctrl.$scope.stage"></kubernetes-manifest-selector>
```

Comment:
s/ctrl.$scope.stage/stage in this template.

Reply:
Or in the controller assign `this.stage = this.$scope.stage` and use `ctrl.stage` in the template.